tock/tock/settings/dev.py

```diff
@@ -54,6 +54,8 @@ def show_django_debug_toolbar(request):
     'SHOW_TOOLBAR_CALLBACK': 'tock.settings.dev.show_django_debug_toolbar',
 }
 
+USE_X_FORWARDED_HOST = True
```

Comment:
I think it's fine to move forward with this as it is.  The instructions are written so that you get yourself setup with Docker if you haven't already.